### PR TITLE
Feature refactore chef profile page

### PR DESCRIPTION
**Pull request by MinaKhamesi**
Bug with regular user experience with chefprofile resolved. if statement for preventing console log err in action added. unnecessary console logs deleted @Vaibhav-Kambli @rudream

### CODE REVIEW

**Comment by rudream** [comment]
Looks good, though I think you should update the chef profile page so that the add to cart button doesn't render if the user viewing the profile is a chef. If I'm a chef looking at a chef profile, there shouldn't be the "add to cart" button, only customers should have carts.

**Comment by MinaKhamesi** [comment]
> Looks good, though I think you should update the chef profile page so that the add to cart button doesn't render if the user viewing the profile is a chef. If I'm a chef looking at a chef profile, there shouldn't be the "add to cart" button, only customers should have carts.

I can make that change @rudream , it will be a conditional rendering of the button and it is done. But I think it will be okay if a chef decided to order from another chef for an event. Maybe they are busy doing their jobs and want to order from their colleagues?

File: client/src/pages/ChefProfile.js [hunk 1]
```diff
@@ -26,13 +26,12 @@ const ChefProfile = ({ history, match }) => {
   const [isOwner, setIsOwner] = useState(true);
 
   useEffect(() => {
-    if (userInfo && userInfo.isChef) {
+    if (userInfo) {
```
Comment (Vaibhav-Kambli) [inline client/src/pages/ChefProfile.js:29]: This is working well with the user profile now, looks good!

**Comment by rudream** [comment]
> > Looks good, though I think you should update the chef profile page so that the add to cart button doesn't render if the user viewing the profile is a chef. If I'm a chef looking at a chef profile, there shouldn't be the "add to cart" button, only customers should have carts.
> 
> 
> 
> I can make that change @rudream , it will be a conditional rendering of the button and it is done. But I think it will be okay if a chef decided to order from another chef for an event. Maybe they are busy doing their jobs and want to order from their colleagues?

I like the idea but that will cause issues later based on how we have the order history pages set up, since the /orders page will look different depending on whether you are a customer or a chef. /orders for a chef shows all the incoming orders, and /orders for a customer shows the orders that customer has made. Unless we want to add a section to the chef /orders page called "My orders"?

**Review by Vaibhav-Kambli — APPROVED** [review]
Only suggestion will be to closing the cart modal on click of proceed, since it stays open at the payment page. This can be done in the future application clean up.

**Review by rudream — APPROVED** [review]
Looks good and the bug seems to be fixed on my tests. You can add my suggestion in a later PR.

**Pull request merged**